test/remote/show_dispatch_test.exs

```diff
@@ -1,7 +1,7 @@
 defmodule Remote.ShowDispatchTest do
   use Remote.Environment.Case
   @moduletag remote: false
-  @moduletag :dispatch
+  @moduletag dispatch: true
```

Comment:
Do you think it would be less confusing to leverage the `remote` moduletag here instead? Looks like we already excluded remote tests by default so why make `dispatch` a special case?

Reply:
The intention of the previous dev was to exclude `dispatch` from the regular remote test suite unless specifically enabled. This goes back to Dispatch not being a feature we plan to carry forward and that this feature must be enabled on the environment. What happened with the implementation is that we accidentally turned Dispatch tests on whenever running a plain `mix test`. The action here is to move back to the original intended state. 

By including `Remote.Environment.Case`, `@moduletag remote: true` is already set. `@moduletag remote: false` overrides this so these tests are in the remote test suite. `@moduletag dispatch: true` let's is run Dispatch tests by themselves.

Reply:
Hmmm if including `Remote.Environment.Case` sets `remote: true` doesn't that achieve the same goal of not running dispatch tests? If I'm follow correctly (which I may not be), we wouldn't need the `moduletag dispatch:` or `moduletag remote` override.

Reply:
It does, but it also includes them running the regular remote tests. What we want is to always exclude Dispatch tests, unless otherwise explicitly included.

`mix test --include remote:true` would include Dispatch tests, we don't want that. We want to run them only when `mix test --only dispatch:true`.

Reply:
Per our chat, I'm going to remove the reference to `dispatch`. Any API requests exposed by this published library should be tested.